### PR TITLE
support enabling and disabling reduced motion

### PR DESCRIPTION
**Pull request by thisconnect**

### CODE REVIEW

File: frontends/web/src/utils/animation.ts [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
+    }
+}
+
 export function animate(
     el: Element,
     effect: string,
     callback?: () => void,
 ) {
+    if (reduceMotion) {
+        if (callback) {
```
Comment (x1ddos) [inline frontends/web/src/utils/animation.ts:22]: `if (reducedMotion && callback) { callback(); }`?

Comment (thisconnect) [reply frontends/web/src/utils/animation.ts:22]: Reduced motion can be true, but without a callback, then we want to stop at the `return` that comes later.

Comment (thisconnect) [reply frontends/web/src/utils/animation.ts:22]: But I could write it different let me know

Comment (x1ddos) [reply frontends/web/src/utils/animation.ts:22]: :see_no_evil: sorry, ignore this.

File: frontends/web/src/utils/animation.ts [hunk 2]
```diff
@@ -1,8 +1,29 @@
...
+            callback();
+        }
+        return;
```
Comment (thisconnect) [inline frontends/web/src/utils/animation.ts:25]: here, if reduceMotion is true we don't want all that animationend that follows and stop with the return.

**Comment by x1ddos** [comment]
Let me try this on qt-linux.

**Comment by x1ddos** [comment]
Nice. Seems to work on qt-linux and android. Let's merge into the main PR and try on macOS.

**Pull request merged**